ReText/editor.py

```diff
@@ -334,6 +342,28 @@ def installFakeVimHandler(self):
 			fakeVimEditor.setQuitAction(self.parent.actionQuit)
 			self.parent.actionFakeVimMode.triggered.connect(fakeVimEditor.remove)
 
+	def updateTextStatistics(self):
+		if not globalSettings.documentStatsEnabled:
+			return
+		start = time.clock()
+		text = self.toPlainText()
+		wasWordCharacter = False
+		wordCount = 0
+		alphaNumCount = 0
+		characterCount = 0
+		for c in text:
+			characterCount += 1
+			isWordCharacter = c.isalnum()
+			if isWordCharacter:
+				alphaNumCount += 1
+			if wasWordCharacter and not isWordCharacter:
+				wordCount += 1
+			wasWordCharacter = isWordCharacter
+		end = time.clock()
+		value = int((end - start) * 1000)
```

Comment:
Are these start / end / value lines a leftover debug? Can you please share your time measurements?